.github/workflows/dev_frontend_pr.yml

```diff
@@ -0,0 +1,59 @@
+name: Dev FE Pull Request
+
+on:
+  pull_request:
+    branches:
+      - development
+    paths:
+      - 'frontend/**'
+
+env:
+  TERRAFORM_VERSION: "1.5.7"
+  TF_IN_AUTOMATION: "True"
+  ENVIRONMENT: "dev"
+  WORKING_DIR: "frontend"
+  DOCKER_FILE: "Dockerfile.azure"
+
+jobs:
+  build:
+    permissions:
+      contents: read
+      packages: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build and publish the Docker image for ${{ github.repository }}
+        uses: macbre/push-to-ghcr@v13
```

Comment:
Prior we used `docker/build-push-action` - it did have a bit more dependency on building from `docker buildx` - just curious if the `macbre` action brings anything different / benefits?